<?xml version="1.0" ?>
<launch>

<arg name="robot_name" default="j2n6s300"/>

<group ns="$(arg robot_name)">

  <!-- Load planner configurations from YAML file to parameter server -->
  <rosparam file="$(find jaco_inv_dynamics_controller)/config/joint_trajectory_controller.yaml" command="load"/>

  <!--launches planner node-->
  <node pkg="jaco_inv_dynamics_controller" type="joint_PID_planner_arm" name="joint_planner" output="screen"/>

</group>

</launch>
